packages/angular/angular.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "../.././node_modules/@angular/cli/lib/config/schema.json",
```

Comment:
does this have to be a relative reference to node_modules?

Reply:
Not in the sense that it would let compilation fail but VSCode doesn't find the schema when only specifying `@angular/cli/lib/config/schema.json`. Is there something different I should try?

Reply:
No just wondering if there would be problems when the user consumes this package - but I guess we don't include `angular.json` in the final npm asset distribution right?

Reply:
nope, that's only used for configuring the angular project/compiler